src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/GoboGraphQLContextFactory.kt

```diff
@@ -34,7 +33,7 @@ class GoboGraphQLContextFactory : SpringGraphQLContextFactory<SpringGraphQLConte
             token = (serverRequest.headers().firstHeader(HttpHeaders.AUTHORIZATION)?.removePrefix("Bearer ") ?: "")
             securityContext = getSecurityContext()
             request = serverRequest
-            startTime = LocalDateTime.now()
+            startTime = System.currentTimeMillis()
```

Comment:
Hvorfor går du bort fra LocalDateTime og tidssone ? Vi trenger kanskje ikke det ?

Reply:
Denne må jeg undersøke mer, jeg fant faktisk en mismatch mellom hvor vi startet tiden (denne koden du ser her) og koden som regnet ut hvor lang tid det har tatt. Den ene var LocalDateTime den andre var long. For meg så det ut som dette var eneste sted den ble brukt, og siden vi regner ut tid brukt i millisekunder så endret jeg denne. Men den her må jeg verifisere.